web/modules/custom/mymodule/mymodule.links.menu.yml

```diff
@@ -0,0 +1,6 @@
+mymodule.simplecontentpage:
+  title: 'Simple Content Link'
+  description: 'Link to the simple content method'
+  route_name: 'mymodule.simpleconte'nt
```

Comment:
Quotation unclosed

Reply:
@Vishalprasad28 How was this code working then?
Did you test after making the changes?

Reply:
@Vishalprasad28 You have not replied to this comment.